### ISSUE TITLE
Symfony 5 / remove autowiring for Controller Services

### ISSUE THREAD

**Issue by tacman** [kevinpapst/AdminLTEBundle#152]
Symfony 5.1 throws a deprecation error for autowiring the controllers as services.  Removing this section and explicitly injecting the services as parameters throws an error that I can't figure out how to handle.

remove this:

```yaml
services:

    KevinPapst\AdminLTEBundle\Controller\:
        resource: '../../Controller'
        tags: ['controller.service_arguments']
        autowire: true
```

replace with this:

```yaml
    KevinPapst\AdminLTEBundle\Controller\EmitterController:
        class: KevinPapst\AdminLTEBundle\Controller\EmitterController
        tags: ['controller.service_arguments']
        arguments:
            - '@event_dispatcher'
            -
    KevinPapst\AdminLTEBundle\Controller\NavbarController:
        class: KevinPapst\AdminLTEBundle\Controller\NavbarController
        tags: ['controller.service_arguments']
        arguments:
            - '@event_dispatcher'
            - '@admin_lte_theme.context_helper'

    KevinPapst\AdminLTEBundle\Controller\SidebarController:
        class: KevinPapst\AdminLTEBundle\Controller\SidebarController
        tags: ['controller.service_arguments']
        arguments:
            - '@event_dispatcher'
```

Gets this:

An exception has been thrown during the rendering of a template (""KevinPapst\AdminLTEBundle\Controller\NavbarController" has no container set, did you forget to define it as a service subscriber?").

But when I tag it as a service subscriber, I still get an error when trying to render the controller.  My Symfony skills are pretty good, but this one is a bit beyond me.  I think it has something to do with passing the arguments as an array:

https://symfony.com/doc/current/service_container/service_subscribers_locators.html#defining-a-service-locator

Since fixing this will work for version 4 as well, can you take a quick look at at point me in the right direction, and I'll submit a PR.  Thanks.

**Comment by kevinpapst** [comment]
Maybe the YAML has a parsing problem because of the empty line above the controller definition?
```
            -
    KevinPapst\AdminLTEBundle\Controller\NavbarController:
```

**Comment by kevinpapst** [comment]
will be fixed with the new major release #109
